packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -58,20 +59,6 @@ export class DocumentStorageService implements IDocumentStorageService {
         }));
     }
 
-    public async read(blobId: string): Promise<string> {
-        const value = await this.manager.getBlob(blobId);
```

Comment:
Everywhere where you remove read(), you need to do assessment if readBlob() has equivalent support to what read() did.
Here, read() was populating this.blobsShaCache, It looks like it is used only as a set, telling if a given sha1 is know to storage, to optimize writing out summaries. So similar logic should be moved to readBlob().
To some extent that's existing bug in code, but by pushing everyone to use readBlob(), we are exposing these bugs.

Please inspect other places where you remove read() for parity in logic between readBlob() and read(), Most likely if there is no parity, then readBlob() is buggy and needs to be fixed